app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ApplicationRecord
+  has_many :reservations, dependent: :destroy
+end

```

Comment:
- [ ] I Think one user can have many cars, you can add association between user and car `has_many :cars, dependent: :destroy`
- [ ] The user can have the image attached, so we can say `has_one_attached :person_image or :user_image`